query-results/template.yaml

```diff
@@ -38,6 +38,33 @@ Resources:
       VersioningConfiguration:
         Status: Enabled
 
+  QueryResultsBucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref QueryResultsBucket
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: AthenaLambdaAccess
+            Action:
+              - s3:GetBucketLocation
+              - s3:GetObject
+              - s3:ListBucket
+              - s3:ListBucketMultipartUploads
+              - s3:ListMultipartUploadParts
+              - s3:AbortMultipartUpload
+              - s3:PutObject
+            Effect: Allow
+            Resource:
+              - '{{resolve:ssm:QueryResultsBucketArn}}'
```

Comment:
I noticed Andy switching from SSM parameters to `!Sub arn:aws:s3:::txma-qr-infra-${Environment}-query-results-bucket` in this PR
https://github.com/alphagov/di-txma-ticf-integration/pull/143/files

Do we need to do the same here or have I got that wrong @andylangridge ?

Reply:
Nope because this stack is deployed to the same account (where the SSM param is accessible). In the other stack (the one in that PR) it won't be there, hence why I hard-coded